.github/workflows/react-on-issue-labels.yml

```diff
@@ -29,3 +30,19 @@ jobs:
           Having any questions or issues? Feel free to ask here on GitHub. Need help setting up your local workspace? Join the conversation on [JabRef's Gitter chat](https://gitter.im/JabRef/jabref). And don't hesitate to open a (draft) pull request early on to show the direction it is heading towards. This way, you will receive valuable feedback.
 
           Happy coding! 🚀
+    - name: Move Issue to "Assigned" Column in "Candidates for University Projects"
+      uses: m7kvqbe1/github-action-move-issues@v1.0.0
```

Comment:
Will it always move or only if it's assigned to that project already?

Reply:
We need to try ^^